plugins/system_upgrade.py

```diff
@@ -511,6 +511,24 @@ def run_reboot(self):
         reboot()
 
     def run_download(self):
+        #  downstream hack to narrow missing upgrade path with modules between dist versions
+        if self.opts.command not in ['offline-upgrade', 'offline-distrosync']:
+            if dnf.base.WITH_MODULES:
+                module_container = self.base._moduleContainer
+                enabled_module_names = set()
+                for module in module_container.getModulePackages():
+                    if module_container.isEnabled(module):
+                        enabled_module_names.add(module.getName())
```

Comment:
The line and line 524 do the same operation

Reply:
Ok, I will remove this check.